scripts.d/21-ssh.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Setup script error handling see https://disconnected.systems/blog/another-bash-strict-mode for details
+set -uo pipefail
+trap 's=$?; echo "$0: Error on line "$LINENO": $BASH_COMMAND"; exit $s' ERR
+IFS=$'\n\t'
+
+# Enable ssh server
+touch /boot/ssh
+
+# Disable password authentication
+#sed -i 's/\#PasswordAuthentication yes/PasswordAuthentication no/' /etc/ssh/sshd_config
+
+# Add public keys
```

Comment:
We should still add the authorized keys, even if we disable the password authentication.

Reply:
The preview is a bit confusing. I just mean line 15 and following. Keep line 12 commented.